### ISSUE TITLE
CI fails on Go 1.16 due to go-acc installation

### ISSUE THREAD

**Issue by at-wat** [pion/.goassets#60]
`go get -u github.com/ory/go-acc` updates go.mod and go.sum.
After installing `go-acc`, `go test` fails by
```
/go/pkg/mod/github.com/stretchr/testify@v1.7.0/assert/assertions.go:20:2: missing go.sum entry for module providing package github.com/davecgh/go-spew/spew (imported by github.com/stretchr/testify/assert); to add:
	go get github.com/stretchr/testify/assert@v1.7.0
FAIL	github.com/pion/interceptor/internal/test [setup failed]
```

**Comment by at-wat** [comment]
Affected PRs:
https://github.com/pion/turn/pull/208
https://github.com/pion/transport/pull/133
https://github.com/pion/srtp/pull/140
https://github.com/pion/sdp/pull/102
https://github.com/pion/rtcp/pull/92
https://github.com/pion/interceptor/pull/31
https://github.com/pion/ice/pull/340

**Comment by Sean-Der** [comment]
Fixed with c207eea76165de7fc0ad768277a054a591ddb0a3
